src/components/profile/edit/account/index.jsx

```diff
@@ -21,32 +23,36 @@ export const AccountProfileEdit = ({ account, toggleVisibleSubstrate, openVisibl
 
   const toggleVisivbleField = (field) => {
     if (!field || field === visivbleField) {
-      setVisivbleField()
+      setVisivbleField();
```

Comment:
Better pass a parameter explicitly in both cases to avoid ambiguities

Reply:
Done